clusterctl/README.md

```diff
@@ -0,0 +1,96 @@
+# 
+# Deploy a mgmt and a workload cluster with clusterctl in kind
+# 
+# This README is based on, read them before continuing
+https://cluster-api.sigs.k8s.io/clusterctl/overview.html
+https://github.com/Nordix/airship-clusterapi-docs/blob/clusterctl-kashif/clusterCtl-study/testcmd/instrusctions.md
+
+# create a cluster on localhost
+# 
+~$ kind create cluster
+
+# get kubeconfig of new kind cluster
+# 
+~/.cluster-api$ kind get kubeconfig > kubeconfig
+
+# copy files for cluster-template.yaml and metadata.yaml
+# 
+cp ~/work/go/src/sigs.k8s.io/cluster-api-provider-metal3/examples/clusterctl-templates/clusterctl-cluster.yaml ~/.cluster-api/overrides/infrastructure-metal3/v0.3.0/cluster-template.yaml
+
+cp ~/work/go/src/sigs.k8s.io/cluster-api-provider-metal3/metadata.yaml ~/.cluster-api/overrides/infrastructure-metal3/v0.3.0/metadata.yaml
+
+~/.cluster-api$ tree
+.
+├── clusterctl.yaml
+├── kubeconfig
+└── overrides
+    ├── bootstrap-kubeadm
+    │   └── v0.3.0
+    │       └── bootstrap-components.yaml
+    ├── cluster-api
+    │   └── v0.3.0
+    │       └── core-components.yaml
+    ├── control-plane-kubeadm
+    │   └── v0.3.0
+    │       └── control-plane-components.yaml
+    ├── infrastructure-baremetal
+    │   └── v0.3.0
+    │       ├── cluster-template.yaml
+    │       ├── infrastructure-components.yaml
+    │       └── metadata.yaml
+    └── infrastructure-metal3
+        └── v0.3.0
+            ├── cluster-template.yaml
+            ├── infrastructure-components.yaml
+            └── metadata.yaml
+
+# clusterctl local overrides generated from local repositories for the cluster-api, bootstrap-kubeadm, control-plane-kubeadm, infrastructure-metal3 providers.
+# in order to use them, please run:
+~/work/go/src/sigs.k8s.io/cluster-api$ 
+cmd/clusterctl/hack/local-overrides.py
```

Comment:
this works only if you have a clusterctl-settings.json defined. You seem to be missing it in the doc

Reply:
done

Reply:
Yes a clusterctl-settings.json is needed both in CAPI and CAPM3 with different content. You may also add the contents and describe it here. You might find it in my PR. Since you are doing it as an How to, I think this will be a good place to add everything. So take any text or file you need from my PR. Once your PR has everything, I might delete mine.

Reply:
done